ddtrace/span.py

```diff
@@ -99,7 +99,7 @@ def __init__(
         self._parent = None
 
         # state
-        self._finished = False
+        self.finished = False
```

Comment:
any value in making this a read only property? not sure why someone would manually set `span.finished = True`, but.. never know

Reply:
It boils down to a documentation feature IMHO.

There's a `finish()` method that indicates that's how you finish the `Span`, to me that's enough to know how to properly start/stop a `Span`.

For example, `duration` is not read-only and if you modify it, it'll have no effect and be erased.

I tend to think that API doc is more useful than trying to set the various properties read-only (that can be bypassed in the end anyhow).

That's my Python philosophy at least 😁

Reply:
👍